docs/src/rules/camelcase.md

```diff
@@ -177,11 +177,11 @@ Examples of **correct** code for this rule with the `{ "ignoreDestructuring": tr
 ```js
 /*eslint camelcase: ["error", {ignoreDestructuring: true}]*/
 
-var { category_id } = query;
+const { category_id } = query;
 
-var { category_id = 1 } = query;
+const { category_name = 1 } = query;
 
-var { category_id: category_id } = query;
+const { category_id: category_id_name } = query;
```

Comment:
```suggestion
const { category_id_name: category_id_name } = query;
```

It needs to be the same name for this example to be correct.

![image](https://github.com/user-attachments/assets/f9cc35dc-d8af-4f6c-8c80-01c99a13ec0c)